sllurp/llrp.py

```diff
@@ -764,6 +764,22 @@ def send_SET_READER_CONFIG(self, onCompletion):
                 'Type': 3,
                 'ID':   0,
                 'ResetToFactoryDefaults': False,
+                'ReaderEventNotificationSpec': {
+                    'EventNotificationState': {
+                            'HoppingEvent': False,
+                            'GPIEvent': False,
+                            'ROSpecEvent': False,
+                            'ReportBufferFillWarning': False,
+                            'ReaderExceptionEvent': False,
+                            'RFSurveyEvent': False,
+                            'AISpecEvent': False,
+                            'AISpecEventWithSingulation': False,
+                            'AntennaEvent': False,
+                            ## Next one will only be available
+                            ## with llrp v2 (spec 1_1)
```

Comment:
Do you have a URL for this?  I am a little confused by the `(spec 1_1)` comment.

Reply:
What i'm speaking about is the llrp spec 1.1:
https://www.gs1.org/sites/default/files/docs/epc/llrp_1_1-standard-20101013.pdf
Its complicated because it is the specification 1.1 instead of 1.0.1, but that defines the protocol v2.
Some things has to be done with version negotiation and all to support the few additions of spec 1.1 but still I added a few things to support that in advance.